jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/HBaseIntegrationTest.java

```diff
@@ -36,404 +34,492 @@ protected JmxScraperContainer customizeScraperContainer(
+                    .hasDataPointsWithOneAttribute(attributeWithAnyValue("region_server")))
+        .add(
+            "hbase.region_server.operation.increment.latency.median",
+            metric ->
+                metric
+                    .isGauge()
+                    .hasDescription("Increment operation median latency.")
+                    .hasUnit("ms")
+                    .hasDataPointsWithOneAttribute(attributeWithAnyValue("region_server")))
+        // -----------------------------------------------------------------------------------
+
+        .add(
+            "hbase.region_server.operations.slow",
+            metric ->
+                metric
+                    .isUpDownCounter()
+                    .hasDescription("Number of operations that took over 1000ms to complete.")
+                    .hasUnit("{operation}")
+                    .hasDataPointsWithAttributes(
+                        attributeSet(
+                            attribute("operation", "delete"),
+                            attributeWithAnyValue("region_server")),
+                        attributeSet(
+                            attribute("operation", "append"),
+                            attributeWithAnyValue("region_server")),
+                        attributeSet(
+                            attribute("operation", "get"), attributeWithAnyValue("region_server")),
+                        attributeSet(
+                            attribute("operation", "put"), attributeWithAnyValue("region_server")),
+                        attributeSet(
+                            attribute("operation", "increment"),
+                            attributeWithAnyValue("region_server"))))
+        .add(
+            "hbase.region_server.open_connection.count",
+            metric ->
+                metric
+                    .isUpDownCounter()
+                    .hasDescription("The number of open connections at the RPC layer.")
+                    .hasUnit("{connection}")
+                    .hasDataPointsWithOneAttribute(attributeWithAnyValue("region_server")))
+        .add(
+            "hbase.region_server.active_handler.count",
+            metric ->
+                metric
+                    .isUpDownCounter()
+                    .hasDescription("The number of RPC handlers actively servicing requests.")
+                    .hasUnit("{handler}")
+                    .hasDataPointsWithOneAttribute(attributeWithAnyValue("region_server")))
+        .add(
+            "hbase.region_server.queue.request.count",
+            metric ->
+                metric
+                    .isUpDownCounter()
+                    .hasDescription("The number of currently enqueued requests.")
+                    .hasUnit("{request}")
+                    .hasDataPointsWithAttributes(
+                        attributeSet(
+                            attribute("state", "replication"),
+                            attributeWithAnyValue("region_server")),
```

Comment:
[for reviewer] Thanks to new attribute assertion logic I discovered that in original implementation region_server was not checked!